ckan/lib/dictization/model_dictize.py

```diff
@@ -388,7 +388,7 @@ def get_packages_for_this_group(group_, just_the_count=False):
 
     image_url = result_dict.get('image_url')
     result_dict['image_display_url'] = image_url
-    if image_url and not image_url.startswith('http'):
```

Comment:
this line of code was already wrong (`startswith('http')` is not how you check for a valid url).  Would you change this to use an existing library to check for a valid url instead?

Reply:
Yes, sure, I'll have a look at it.

Reply:
Does it suit you this way ?